chain/client/src/stateless_validation/partial_witness/partial_witness_actor.rs

```diff
@@ -425,7 +455,20 @@ impl PartialWitnessActor {
                     return Ok(());
                 }
             };
-            // TODO(#11099): precompile contracts using self.runtime
+            let runtime = self.runtime.clone();
+            self.compile_contracts_spawner.spawn("precompile_deployed_contracts", move || {
+                match runtime.precompile_contracts(&key.epoch_id, contract_codes) {
```

Comment:
nit: `if Err(err) = runtime.precompile_contracts(..)` instead of match

Reply:
done